File: src/WorkBC.Data/Model/JobBoard/SavedIndustryProfile.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace WorkBC.Data.Model.JobBoard
{
    public class SavedIndustryProfile
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        [Column("IndustryId")]
        public short IndustryId { get; set; }

        [ForeignKey("JobSeeker")]
        public string AspNetUserId { get; set; }

        public virtual JobSeeker JobSeeker { get; set; }

        public DateTime DateSaved { get; set; }
        public DateTime? DateDeleted { get; set; }
        public bool IsDeleted { get; set; } = false;
    }
}

File: app/Models/SavedIndustryProfile.php
<?php

namespace App\Models;

use App\Models\Concerns\SoftDeletesByFlag;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

/**
 * ACCT-6 — a job seeker's saved industry profile.
 *
 * `IndustryId` is a smallint FK to `Industries`. Note the legacy list joins
 * `Industries.TitleBC` (the B.C.-specific title), not `Title` — see
 * {@see \App\Services\JobSeeker\SavedProfileService::industryProfilesFor()}.
 */
final class SavedIndustryProfile extends Model
{
    use SoftDeletesByFlag;

    protected $table = 'SavedIndustryProfiles';

    protected $primaryKey = 'Id';

    protected $keyType = 'int';

    public $incrementing = true;

    public $timestamps = false;

    protected $casts = [
        'IsDeleted' => 'boolean',
        'IndustryId' => 'integer',
        'DateSaved' => 'datetime',
        'DateDeleted' => 'datetime',
    ];

    public function jobSeeker(): BelongsTo
    {
        return $this->belongsTo(JobSeeker::class, 'AspNetUserId', 'Id');
    }

    public function industry(): BelongsTo
    {
        return $this->belongsTo(Industry::class, 'IndustryId', 'Id');
    }
}
